app/src/main/java/com/ably/tracking/demo/publisher/api/OrderResponse.kt

```diff
@@ -0,0 +1,10 @@
+package com.ably.tracking.demo.publisher.api
+
+data class OrderResponse(
+    val to: Destination
```

Comment:
I'd change this variable name to `destination` or something similar :thinking: 
It makes sense to call the destination `to` (so we can have `from` and `to`) but when you use this in code it looks a bit weird :sweat_smile: For example when I saw `response.to` my first thought was "to what? to string, to int, to domain model?" :grin: